### PR TITLE
participants controller spec

### PR DESCRIPTION
**Pull request by oreoluver**
addresses #15

### CODE REVIEW

File: spec/features/participant_addition_spec.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      click_on 'Add'
+      expect(page).to have_text 'You are not authorized to make that action.'
+    end
+  end
+  context 'that is a duplicate' do
+    it 'will not add a participant' do
+      when_current_user_is :admin
+      create :participant, name: 'Foo Bar'
+      visit participants_url
+      within('tr:nth-child(2)') do
```
Comment (dfaulken) [inline spec/features/participant_addition_spec.rb]: So if we move the page elements around, will this test break?

**Comment by dfaulken** [comment]
Don't forget to run `rubocop`.

File: spec/features/participant_update_spec.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    end
+  end
+  context 'with master of ceremonies privilege' do
+    it 'will assign a number' do
+      create :bus, number: 'Big Yellow Bus'
+      when_current_user_is :master_of_ceremonies
+      visit participants_url
+      fill_in 'participant_name', with: 'Foo Bar'
+      click_on 'Add'
+      select('Foo Bar', from: 'id')
```
Comment (dfaulken) [inline spec/features/participant_update_spec.rb]: don't need these parentheses

File: spec/features/participant_update_spec.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe 'updating a participant' do
+  context 'with master of ceremonies privilege' do
+    it 'updates a participant' do
+      create :participant, name: 'Foo Bar'
+      when_current_user_is :master_of_ceremonies
+      visit participants_url
+      within ('tr:nth-child(3)') do
```
Comment (dfaulken) [inline spec/features/participant_update_spec.rb]: Same question - is there a more meaningful page element to attach to?

Also, these parentheses aren't needed.

File: spec/features/participant_remove_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+describe 'removing a participant' do
+  before :each do
+    create :participant
+  end
+  context 'with admin privilege' do
+    it 'removes a participant' do
+      when_current_user_is :admin
+      visit participants_url
+      click_on 'Remove'
+      expect(page).to have_text 'Participant has been removed'
```
Comment (dfaulken) [inline spec/features/participant_remove_spec.rb]: I think this (and the other expected texts) look better with the period at the end.

File: spec/features/participant_addition_spec.rb [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    end
+  end
+  context 'with a unique number and no bus number' do
+    it 'will not add a participant' do
+      when_current_user_is :admin
+      visit participants_url
+      fill_in 'participant_name', with: 'Foo Bar'
+      fill_in 'participant_number', with: '1'
+      click_on 'Add'
+      expect(page).to have_text "Bus can't be blank"
```
Comment (dfaulken) [inline spec/features/participant_addition_spec.rb]: Single quotes please

Comment (AkivaGreen) [inline spec/features/participant_addition_spec.rb]: We use a **'** in the **can't** word. To avoid using a slash.

Comment (dfaulken) [inline spec/features/participant_addition_spec.rb]: Whoops, missed that. You're correct.

**Pull request merged**